abstractions/README.md

```diff
@@ -49,29 +44,3 @@ Supports:
 2. Released versions
     * `MAJOR.MINOR.PATH` where `MAJOR` is still supported as defined by the EOL policy of OpenSearch.
     * Note if the version exists but is not yet released it will resolve as a build candidate
-    
```

Comment:
Curious why this is removed from the readme?

Reply:
It was asked for in the [upstream PR](https://github.com/opensearch-project/opensearch-net/pull/92#discussion_r961923490).